rskj-core/src/main/java/org/ethereum/vm/program/invoke/ProgramInvokeFactory.java

```diff
@@ -37,9 +37,8 @@ ProgramInvoke createProgramInvoke(Transaction tx, int txindex, Block block,
                                       Repository repository, BlockStore blockStore);
 
     ProgramInvoke createProgramInvoke(Program program, DataWord toAddress, DataWord callerAddress,
```

Comment:
Why did you change this? Please either revert or follow the [formatting guidelines](https://github.com/rsksmart/rskj/blob/master/CONTRIBUTING.md#formatting).